core/src/test/java/org/apache/iceberg/TestMetadataTableScans.java

```diff
@@ -462,6 +479,180 @@ public void testDataFilesTableSelection() throws IOException {
     Assert.assertEquals(expected, scan.schema().asStruct());
   }
 
+  @Test
+  public void testDeleteFilesTableScanNoFilter() {
+    Assume.assumeTrue("Only V2 Tables Support Deletes", formatVersion >= 2);
+
+    preparePartitionedTable();
+
+    Table deleteFilesTable = new DeleteFilesTable(table.ops(), table);
+    Types.StructType expected = new Schema(
+        required(102, "partition", Types.StructType.of(
+            optional(1000, "data_bucket", Types.IntegerType.get())),
+            "Partition data tuple, schema based on the partition spec")).asStruct();
+
+    TableScan scanNoFilter = deleteFilesTable.newScan().select("partition.data_bucket");
+    Assert.assertEquals(expected, scanNoFilter.schema().asStruct());
+    CloseableIterable<FileScanTask> tasksAndEq = scanNoFilter.planFiles();
```

Comment:
nit: looks like a copy and paste typo as I am not sure what `tasksAndEq` means. I did not check all tests.

Reply:
Yea you are right, sorry I fixed all the tasks variables across the newly added tests